test/quantization/fx/test_quantize_fx.py

```diff
@@ -1159,7 +1159,7 @@ def forward(self, x):
             "module_name_regex": [("module_conv*", module_name_regex_qconfig)],
             "module_name": [("module_conv2", module_name_qconfig)]}
         m = prepare_fx(m, qconfig_dict)
-        self.assertEqual(m.linear.qconfig, global_qconfig)
+        self.assertEqual(m.linear.qconfig, qconfig_dict[""])
```

Comment:
should we revert this change?

Reply:
yeah reverted

Reply:
had to do some significant finagling to get this test working, It works correctly but the assertEqual kept breaking due to how the m.linear.qconfig would get updated with the new constructor device info. Its working now though.